docs/docs/guides/receiving/filtering.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Subscriptions with filter
+nav_order: 2
+parent: Receiving messages
+grand_parent: How to Guides
+---
+
+When you receive messages from a subscription with a filter, you only receive the messages that match the filter. Metro automatically acknowledges the messages that don't match the filter. You can filter message by their attributes.
+
+# Creating subscriptions with filters
+
+To create a subscription with filter capability, filtering expression need to be passed along with the payload while creating subscriptions in `filter` field.
+
+Sample Subscription:
+```json
+{
+    "topic": "projects/my-project/topics/mytopic",
+    "ackDeadlineSeconds": 90,
+    "retryPolicy": {
+        "minimumBackoff": "5s",
+        "maximumBackoff": "30s"
+    },
+    "deadLetterPolicy": {
+        "maxDeliveryAttempts": 5
+    },
+	"filter": "<Filter Expression>"
+}
+```
+
+Filter can be used in the similar manner while creating the push subscription also.
+
+# Filtering syntax
+
+To filter messages, write an expression that operates on attributes. You can write an expression that matches the key or value of the attributes. The **attributes** identifier selects the attributes in the message.
```

Comment:
nit: The **attributes** identifier provides access to the attributes sent as part of a message.